polaris-migrator/src/utilities/sass.ts

```diff
@@ -489,7 +489,7 @@ export function createSassMigrator(name: string, ruleFn: PolarisMigrator) {
       const {walker, serialiseSuggestion} = args;
 
       return (node: T) => {
-        let oldNode: T;
```

Comment:
TS complained about `node` being instantiated with a different sub type than `oldNode`, but giving it a concrete type here instead of the generic fixed it.